CoreScriptsRoot/Modules/VR/Panel3D.lua

```diff
@@ -152,10 +108,10 @@ function Panel3D.RaycastOntoPanel(part, parentGui, gui, ray)
 	local planeNormal = planeCF.lookVector
 	local pointOnPlane = planeCF.p + (planeNormal * partThickness * 0.5)
 
-	local worldIntersectPoint = Util.RayPlaneIntersection(ray, planeNormal, pointOnPlane)
+	local worldIntersectPoint = Utility:RayPlaneIntersection(ray, planeNormal, pointOnPlane)
 	if worldIntersectPoint then
 		local parentGuiWidth, parentGuiHeight = parentGui.AbsoluteSize.X, parentGui.AbsoluteSize.Y
-		local localIntersectPoint = (planeCF:pointToObjectSpace(worldIntersectPoint) / currentHeadScale) * Vector3.new(-1, 1, 1) + Vector3.new(partWidth / 2, -partHeight / 2, 0)
```

Comment:
Fixes a bug where the cursor's position would essentially be scaled down by the head scale, so HeadScale=10 would result in 1/10th as much movement as necessary.